themes/default/content/product/_index.md

```diff
@@ -12,80 +12,115 @@ subheading: |
-        By using the industry's best languages, you can skip learning proprietary, limited domain-specific
-        languages, and enable your team to work more productively and better together.
+        Ship infrastructure at faster velocity with speed and scale enabled by software engineering and an API-first approach. Pulumi open source enables engineers
+        to build with any cloud using general-purpose programming languages instead of proprietary languages. It’s backed by an enterprise-grade SaaS that increases
+        velocity, collaboration, security for teams and organizations.
 
 key_features:
     title: Key features
     items:
-        - description: Build [Infrastructure as Code](/what-is/what-is-infrastructure-as-code/) in familiar languages
+        - description:
           features:
-              - title: Languages you love
+              - title:
                 icon: code
                 icon_color: yellow
                 description: |
-                    Use TypeScript, JavaScript, Python, Go, .NET, Java, and YAML to model cloud infrastructure
-                    by leveraging the features of each language.
+                    Declarative IaC in TypeScript/JavaScript, Python, Go, C#, Java, and YAML. Deep runtime integrations and no proprietary frameworks.
 
-              - title: Build on any cloud
+              - title:
                 icon: global
                 icon_color: yellow
                 description: |
-                    Access the full breadth of services in AWS, Azure, GCP, and [60+ providers](/registry/) through
-                    a complete and consistent SDK interface.
+                    Use your favorite IDEs with autocomplete, in-line documentation, and a natural development experience.
 
-              - title: Create reusable infrastructure
+              - title:
                 icon: puzzle
                 icon_color: yellow
                 description:
-                    Build and share cloud infrastructure with reusable [Pulumi Packages](/product/packages/), which
-                    encapsulate your cloud architectures and best practices.
+                    Run rapid unit tests with any supported framework to validate your infrastructure code before deployment.
 
-        - description: Deploy cloud infrastructure and applications together
+        - description: Consume infrastructure as code in any language
           features:
-              - title: Multiple deployment options
+              - title:
                 icon: rocketship
                 icon_color: salmon
                 description: |
-                    Deploy infrastructure interactively with a CLI, programmatically with Pulumi’s
-                    [Automation API](/automation/), or through your [CI/CD process](/docs/guides/continuous-delivery/).
+                    Build faster by reusing Pulumi libraries and CDK constructs, and create your own libraries as multi-language Pulumi Packages.
```

Comment:
link CDK constructs to: https://www.pulumi.com/blog/aws-cdk-on-pulumi/
link multi-language Pulumi Packages to https://www.pulumi.com/product/packages/
link Pulumi libraries: https://www.pulumi.com/docs/intro/concepts/resources/components/